web/src/components/TestCard/TestCard.tsx

```diff
@@ -1,40 +1,31 @@
 import {DownOutlined, RightOutlined} from '@ant-design/icons';
-import {Button} from 'antd';
+import {Button, Typography} from 'antd';
 import React, {useCallback, useState} from 'react';
-import {useLazyGetRunListQuery} from '../../redux/apis/TraceTest.api';
-import {TTest} from '../../types/Test.types';
-import ResultCardList from '../RunCardList';
+
+import ResultCardList from 'components/RunCardList';
+import {useLazyGetRunListQuery} from 'redux/apis/TraceTest.api';
+import {TTest} from 'types/Test.types';
 import * as S from './TestCard.styled';
 import TestCardActions from './TestCardActions';
 
-interface TTestCardProps {
-  test: TTest;
-
+interface IProps {
   onClick(testId: string): void;
-
   onDelete(test: TTest): void;
-
   onRunTest(testId: string): void;
+  test: TTest;
 }
 
-const TestCard: React.FC<TTestCardProps> = ({
-  test: {name, serviceUnderTest, id: testId},
-  test,
-  onClick,
-  onDelete,
-  onRunTest,
-}) => {
+const TestCard = ({onClick, onDelete, onRunTest, test: {name, serviceUnderTest, id: testId}, test}: IProps) => {
```

Comment:
Lets use `React.FC`

Reply:
Hey @xoscar thanks for the review. Today `React.FunctionComponent` (or `React.FC`) is discouraged by the community. 

> React.FC is unnecessary: it provides next to no benefits and has a few downsides.  

You can read about this in the following links:


- https://github.com/typescript-cheatsheets/react#function-components
- https://github.com/facebook/create-react-app/pull/8177
- https://kentcdodds.com/blog/how-to-write-a-react-component-in-typescript

Reply:
Sounds good, I think `React.FC` hasn't caused any harm to us. I would suggest introducing these changes in a more global way instead of having two different ways of creating components. We can create a tech debt ticket if this is truly required.

Reply:
Sure thing!